tests/unit/test_data.py

```diff
@@ -73,13 +73,13 @@ def test_dataset_raises_on_initialisation_for_invalid_ranks(
 
 
 def test_dataset_getters() -> None:
-    query_points, observations = tf.zeros((3, 3)), tf.zeros((3, 3))
+    query_points, observations = tf.constant([[0.0]]), tf.constant([[1.0]])
```

Comment:
this wasn't a very good test, as there wasn't any difference between the example query points and observations